### ISSUE TITLE
Need a 'Contributing.md' doc with instructions on how to build

### ISSUE THREAD

**Issue by jazzdelightsme** [AArnott/Nerdbank.GitVersioning#95]
I downloaded a copy of the project and opened the .sln in Visual Studio 2015 (with Update 3). I got a couple of errors that some project types are not supported. Once those were dismissed, it popped up a migration report with some details about the errors:

```
nerdbank-gitversioning.npm\nb-gv.njsproj: The application which this project type is based on was not found. Please try this link for further information: http://go.microsoft.com/fwlink/?LinkId=659044&projecttype=9092AA53-FB77-4645-B42D-1CCCA6BD08BD
```
```
Nerdbank.GitVersioning.NuGet\Nerdbank.GitVersioning.NuGet.nuproj: The application which this project type is based on was not found. Please try this link for further information: http://go.microsoft.com/fwlink/?LinkId=659044&projecttype=FF286327-C783-4F7A-AB73-9BCBAD0D4460
```

Clicking on those links resolved to URLs like: http://dotnetredir.azurewebsites.net/visualstudio?build=Dev14_VSU3_RTW&projecttype=FF286327-C783-4F7A-AB73-9BCBAD0D4460

The pages stuck for a long time on "waiting for response from server", then eventually displayed a simple page with the content "The specified CGI application encountered an error and the server terminated the process."

Does this project require VS2017? Or some special other something installed into VS? It would be great to have a simple "Contributing.md" doc that lists the requirements for building the source.

Thanks!

**Comment by jazzdelightsme** [comment]
(forgot to add that after all that, I tried to build the solution, but it failed with a handful of errors, which I could try to figure out, but I imagine there are just some prereq's I'm missing which are throwing a wrench into the works)

**Comment by AArnott** [comment]
Fixed by 310d235
